cmd/otelcol/config/collector/ecs_ec2_config.yaml

```diff
@@ -151,15 +151,13 @@ service:
       receivers: [prometheus/internal]
       processors: [memory_limiter, batch, resourcedetection/internal]
       exporters: [signalfx]
-#    logs:
-#      receivers: [otlp]
+    logs:
+      receivers: [otlp]
 #      processors:
 #        - memory_limiter
 #        - batch
 #        - resourcedetection
```

Comment:
We also need to enable these processors, they are enabled by default for any other data types

Reply:
Got it, makes sense. Just updated the PR with that change.